docs/getting-started-guides/coreos/azure/index.md

```diff
@@ -1,246 +0,0 @@
----
```

Comment:
Hm, I don't have a strong opinion, but I would hesitate to point people to the weaveworks-guide unless it's something you plan to support (and I'm guessing it doesn't have the cloudprovider wired up). The supportability aspect is part of the reason I want to remove it; even as of last month people were asking questions about it that I couldn't necessarily answer.

I'd be less hesitant about preserving the URL and putting a note here that encourages people to seek out an alternative Azure method. (Though none of the "current" bring-up methods support CoreOS or Weave...)